### ISSUE TITLE
Classic 505 doesn't have

### ISSUE THREAD

**Issue by gaw19** [mylesgray/homebridge-blueair#13]
When using this plugin with classic 505 without sensor, it throws error complaining json content.
I believe it doesn't have any response for this API:

> /v2/device/' + this.deviceuuid + '/datapoint/

I think you did pretty god job in guarding every single measurement pulling with config flag. I think may be you can add another flag "pull_measurements" or something to skip initial pulling in the following block

https://github.com/mylesgray/homebridge-blueair/blob/046808126862b073f81c7d779d2f141428945c31/index.js#L238-L242

**Comment by mylesagray** [comment]
Duplicate of #12 - currently waiting on some JSON output from someone with a Classic so I can get this fixed - see 
https://github.com/mylesgray/homebridge-blueair/issues/12#issuecomment-439107269
